src/core/render/compiler.js

```diff
@@ -19,7 +19,7 @@ export function getAndRemoveConfig(str = '') {
     str = str
       .replace(/^'/, '')
       .replace(/'$/, '')
-      .replace(/:([\w-]+)=?([\w-]+)?/g, (m, key, value) => {
+      .replace(/:([\w-]+)=?([\w-]+)?/, (m, key, value) => {
```

Comment:
This is for extracting all possible configs. In the case of headings, we have only one at the moment. 

I guess a better solution would be only match strings at the end. We may need to introduce a delimiter  for determining where the starting point of configs is.